solr/core/src/java/org/apache/solr/handler/GraphHandler.java

```diff
@@ -87,14 +88,10 @@ public void inform(SolrCore core) {
     }
 
     // This pulls all the overrides and additions from the config
-    Object functionMappingsObj = initArgs.get("streamFunctions");
```

Comment:
I wonder with backwards compatibility in mind, if for a period of time `streamFunctions` might be deprecated but still supported alongside the new `expressible` equivalent. Users would see the 'deprecated' warnings being logged and could transition callers ahead of `streamFunctions` support being taken away in a later version. What do you think?

Reply:
I wondered the same thing.   I don't think anyone actually customizes this handler, but that is just a guess.   I can add a deprecated warning, do you have a good example that I could copy of how we normally do that?

Reply:
@cpoerschke I am pushing up a fix to do the deprecations...